src/base/ApplicationECSService.spec.ts

```diff
@@ -243,6 +243,20 @@ describe('ApplicationECSService', () => {
     expect(synthed).toMatchSnapshot();
   });
 
+  it('renders an ECS service with code deploy notifications set for failed only', () => {
```

Comment:
For this test, I'm using the same snapshot from above. It seems to pass so I'm assuming there's no change to it when creating the ECS service object?